test/debug_naming_tests.cc

```diff
@@ -0,0 +1,65 @@
+#include "task.h"
+#include "task_manager.h"
+#include "types.h"
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_string.hpp>
+
+#include <celerity.h>
+
+#include "test_utils.h"
+
+namespace celerity {
+namespace detail {
+	TEST_CASE_METHOD(test_utils::runtime_fixture, "tasks debug names", "[debug]") {
+		const std::string task_name = "sample task";
+
+		auto tt = test_utils::task_test_context{};
+
+		SECTION("Host Task") {
+			const auto tid_a = test_utils::add_host_task(tt.tm, on_master_node, [&](handler& cgh) {
+				celerity::debug::set_task_name(cgh, task_name);
+			});
+
+			const auto tid_b = test_utils::add_host_task(tt.tm, on_master_node, [&](handler& cgh) {});
+
+			const auto its_a = tt.tm.get_task(tid_a)->get_debug_name();
+			const auto its_b = tt.tm.get_task(tid_b)->get_debug_name();
+
+			CHECK_THAT(its_a, Catch::Matchers::Equals(task_name));
+			CHECK_THAT(its_b, Catch::Matchers::Equals(""));
```

Comment:
```suggestion
			CHECK(its_a == task_name);
			CHECK(its_b.empty());
```

same for all other uses of `Catch::Matchers::Equals` in these tests.

Reply:
fixed